src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -2980,16 +2979,19 @@ namespace PackagedTerminalHeatPump {
 			MySizeFlag( PTUnitNum ) = false;
 		}
 
+		RhoAir = StdRhoAir;
```

Comment:
@rraustad I also noticed that the mass flow rate through this object uses standard air assumptions. It does not use psychrometric functions on the timestep basis to determine the density of air. This doesn't seem quite right to me. What do you think?